DOCUMENTATION.md

```diff
@@ -734,11 +698,56 @@ If the request is not successful, the client returns an `HTTPError` containing t
 |`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Invalid token: API key not found"`<br>`}]`|Use the correct API key. Refer to [API keys](#api-keys) for more information|
 
 
-## Get a PDF for a letter notification
+## Email status descriptions
+
+|Status|Description|
+|:---|:---|
+|`created`|GOV.UK Notify has placed the message in a queue, ready to be sent to the provider. It should only remain in this state for a few seconds.|
+|`sending`|GOV.UK Notify has sent the message to the provider. The provider will try to deliver the message to the recipient for up to 72 hours. GOV.UK Notify is waiting for delivery information.|
+|`delivered`|The message was successfully delivered.|
+|`permanent-failure`|The provider could not deliver the message because the email address was wrong. You should remove these email addresses from your database.|
+|`temporary-failure`|The provider could not deliver the message. This can happen when the recipient’s inbox is full. You can try to send the message again.|
+|`technical-failure`|Your message was not sent because there was a problem between Notify and the provider.<br>You’ll have to try sending your messages again.|
+
+## Text message status descriptions
+
+|Status|Description|
+|:---|:---|
+|`created`|GOV.UK Notify has placed the message in a queue, ready to be sent to the provider. It should only remain in this state for a few seconds.|
+|`sending`|GOV.UK Notify has sent the message to the provider. The provider will try to deliver the message to the recipient for up to 72 hours. GOV.UK Notify is waiting for delivery information.|
+|`pending`|GOV.UK Notify is waiting for more delivery information.<br>GOV.UK Notify received a callback from the provider but the recipient’s device has not yet responded. Another callback from the provider determines the final status of the text message.|
+|`sent` or <br>`sent-internationally`|The message was sent to an international number. The mobile networks in some countries do not provide any more delivery information. The GOV.UK Notify client API returns this status as `sent`. The GOV.UK Notify client app returns this status as `Sent internationally`.|
```

Comment:
is `sent-internationally` an API status?

Reply:
This will need looking into.

It appears that it is correct that - The GOV.UK Notify client API returns this status as `sent`

However, I can't see anything in the python api client at least that transforms that into 'Sent internationally' or 'sent-internationally' but I would like a second opinion on this please as I'm finishing up before I go away and don't have time to double check

Reply:
➕  to changing "The GOV.UK Notify client app".

I'm speaking to CHS about this – as we should come up with a name we can use consistently in the docs, on GOV.UK Notify and amongst ourselves as a team.

Reply:
We've decided on 'GOV.UK Notify website' to describe the UI. This term should be used universally in the docs, on Notify and in our correspondence. I'll update the style guide in our team wiki to reflect this.